frontend/public/module/k8s/k8s-actions.js

```diff
@@ -28,6 +26,7 @@ const POLLs = {};
 const REF_COUNTS = {};
 
 const nop = () => {};
+const paginationLimit = 100;
```

Comment:
What do you think about a higher number? A 10K cluster would require 100 requests to finish loading.  For Pods at least, we show the current count which would tick up ... for a while.  It looks like pods run about 7KB per item - nodes are closer to 12.  A limit of 250 would land at ~2MB per response.